Documentation/Coherency.md

```diff
@@ -0,0 +1,74 @@
+# Problem #
+-----------
```

Comment:
Using both header formats here results in two lines, I suggest just using the `#`s. (I even just use the beginning one, `# Problem`.)

Reply:
Fixed.